[Synthetics] Private Location 1.47.0

lefebvree opened the pull request:
#### What this PR does / why we need it:

Update chart for release 1.47.0 of the Synthetics Private Location.



#### Checklist

- [x] Chart Version bumped
- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] `CHANGELOG.md` has been updated
- [x] Variables are documented in the `README.md`

RodrigoMedeiros85 commented:
CI [failed to push](https://github.com/DataDog/helm-charts/actions/runs/8702337157)  this chart version.  Maybe, run it again with _chart-releaser-action_ `skip_existing ` ?

lefebvree commented:
Hello @RodrigoMedeiros85, thanks a lot for raising this!
We fixed the 0.16.1 release and added a check on our end to ensure future versions are published properly.
